babel.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
```

Comment:
__[HIGH]__  
Merge `babel.config.js` and `babelrc`. There should not exist both of them.